test/libsolidity/syntaxTests/array/concat/bytes_concat_on_variable.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function f() public {
+        bytes memory a;
+        bytes memory b = a.concat();
```

Comment:
Should also add a test on the type information: `type(bytes).concat`.

Reply:
Added test, but it doesn't add that much value right now  as `type(bytes)` errors out with 
`// TypeError 4259: (93-98): Invalid type for argument in the function call. A contract type or an integer type is required, but type(bytes) provided.`

Should I leave test?